docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+.. _kinto-glossary:
+
+Glossary
+########
+
+
+.. glossary::
+
+    Cliquet Protocol
+            A system of rules that explains the way to interact with Kinto,
+            and how its data is organized.
+
+    HTTP API
+            The definition of a set of HTTP request messages along with the
```

Comment:
This is the definition from Wikipedia = https://en.wikipedia.org/wiki/Application_programming_interface#Web_APIs

I am ok to change it, but in "HTTP resources endpoint" , I don't understand what the word resource means.